Source/WebCore/css/process-css-properties.py

```diff
@@ -2761,7 +2761,7 @@ def _generate_css_property_settings_hasher(self, *, to):
         to.write(f"void add(Hasher& hasher, const CSSPropertySettings& settings)")
         to.write(f"{{")
         with to.indent():
-            to.write(f"unsigned bits = {first}")
```

Comment:
Why are we updating this?

Reply:
I was trying to fix the build failure on non-mac platforms, but it seems unsuccessful